setup.cfg

```diff
@@ -1,3 +1,47 @@
+[metadata]
```

Comment:
I'm guessing this entire section has been moved by the package that initially generated it?

Reply:
I manually copied them over from inputs to `setuptools.setup()` following the docs